package.json

```diff
@@ -1,27 +1,27 @@
 {
     "name": "ufds",
-    "version": "7.3.1",
+    "version": "7.3.2",
```

Comment:
Micro bump because from the outside no interfaces have changed, right?  I'm pretty sure I'm correct, but I'm asking for confirmation of my understanding (not because I think it's in error).